packages/vite/src/module-runner/runner.ts

```diff
@@ -54,6 +60,9 @@ export class ModuleRunner {
   ) {
     this.evaluatedModules = options.evaluatedModules ?? new EvaluatedModules()
     this.transport = normalizeModuleRunnerTransport(options.transport)
+    if (options.builtins) {
+      this.isBuiltin = createIsBuiltin(options.builtins)
+    }
```

Comment:
Instead of receiving the options here, I think we should ask the server side by `this.transport.invoke` so that users won't have to set the option.